ci/travis-windows-script.sh

```diff
@@ -3,7 +3,7 @@
 set -e
 
 mkdir build && cd build
-cmake .. -G"Unix Makefiles" -DCMAKE_BUILD_TYPE=Release -DEXTRA_TESTS=OFF
+cmake .. -G"Unix Makefiles" -DCMAKE_BUILD_TYPE=Release -DEXTRA_TESTS=OFF -DCMAKE_UNITY_BUILD=ON
```

Comment:
I would prefer to make an extra build with this flag and normal, just to make sure both works. Linux build is reasonably fast and cheap.